.github/workflows/ci-app-examples.yml

```diff
@@ -6,13 +6,20 @@ on: # Trigger the workflow on push or pull request, but only for the master bran
     branches: [master, "release/*"]
   pull_request:
     branches: [master, "release/*"]
+    paths:
+      - ".github/workflows/ci-app-examples.yml"
+      - "src/lightning_app/**"
+      - "tests/tests_app/**"
+      - "tests/tests_app_examples/**"
+      - "requirements/app/**"
+      - "setup.py"
```

Comment:
Why setup.py?

```suggestion
```

Reply:
The package needs to be installed, so if anything has changed, this is quite rare...